droopler.install

```diff
@@ -2247,3 +2247,15 @@ function droopler_update_8137(&$sandbox) {
   \Drupal::service('d_update')->checklist()
     ->saveProgress(['v3.0.1' => $values]);
 }
+
+/**
+ * Droopler 3.0.1 Updating d_p_tiles paragraph field grouping.
+ */
+function droopler_update_8138(&$sandbox) {
+  $values['droopler__droopler_update_8138_1'] = update_modules_configuration([
+    'd_p_tiles',
```

Comment:
You have to create the update file like in app/web/profiles/contrib/droopler/modules/custom/d_p_side_image/config/update/d_p_side_image.8128.yml

According to https://documentation.droptica.org/node/533